engine/src/cmd/basecomputer.cpp

```diff
@@ -5234,7 +5233,7 @@ void showUnitStats( Unit *playerUnit, string &text, int subunitlevel, int mode,
         text += "#n##n##c0:1:.5#"+prefix+"[DURABILITY STATISTICS]#n##-c";
         text += "#n#"+prefix+statcolor+"Armor damage resistance:#-c";
     }
-    if (mode && MODIFIES(replacement_mode, playerUnit, blankUnit, armor.frontlefttop)) {
+    if (mode && MODIFIES(replacement_mode, playerUnit, blankUnit, armor->facets[0].health)) {
```

Comment:
what does `facets[0]` represent? Using an enum instead of a raw integer would be better at conveying that information

Reply:
It would, but in this case, it really is just any facet. It's doing a check on a single facet and extrapolating the result for all others.

Reply:
Fixed by adding value to enum.